lib/tsd-file-api-client/src/main/java/no/uio/ifi/tc/TSDFileAPIClient.java

```diff
@@ -597,7 +600,7 @@ public TSDFileAPIClient build() {
 
       TSDFileAPIClient tsdFileAPIClient = new TSDFileAPIClient(httpClient);
 
-      tsdFileAPIClient.protocol = this.secure == null ? "https" : (this.secure ? "https" : "http");
+      tsdFileAPIClient.protocol = "http";
```

Comment:
Will this not break https operation of the TSD File API client when running with SSL in operation/staging ?

Reply:
Good catch! thank you, Kjell.

I have resolved this with commit [d7452e9](https://github.com/ELIXIR-NO/FEGA-Norway/pull/189/commits/d7452e937993853b994279fbe7d8b60e960f82a1)

Added the missing `secure()` builder method and added the ternary condition. the default value is now `true` if the value is not set. we override this in the e2eTests setup by passing an ENV variable.